### PR TITLE
updated the tag name of the image

### PR DESCRIPTION
**Pull request by LukeLoze1**
# Motivation and Context
When building locally the image is currently census-rm-casesvc, docker-dev looks for eu.gcr.io/census-rm-ci/rm/census-rm-casesvc.  So having this changes makes using locally easier

# What has changed
Change to tagging in pom.xml

# How to test?
Local copy, mvn clean install.
Then:  docker ps | grep casesvc
The latest image should be eu.gcr.io/census-rm-ci/rm/census-rm-casesvc

# Links
https://trello.com/c/8sVc2ua0/624-fix-built-image-name-for-casesvc

### CODE REVIEW

**Review by hbrace — APPROVED** [review]

**Review by davidmort — APPROVED** [review]

**Comment by AdeHarrison** [comment]
do not merge yet

**Review by AdeHarrison — CHANGES_REQUESTED** [review]
Blocked due to travis build already prepending DOCKER_GCP_LOCATION to the image name, docker tag command fails but push still succeeds. We need an alternative to setting in pom file - maybe just document in docker dev to tag locally built images manually?

**Comment by AdamHawtin** [comment]
@AdeHarrison Is there a reason not to just remove the re-tagging step from the travis script?

File: .travis.yml [hunk 1]
```diff
@@ -32,3 +30,4 @@ cache:
 branches:
   only:
   - master
+  - update-image-tag
```
Comment (AdamHawtin) [inline .travis.yml]: This should only be master?

**Review by AdamHawtin — CHANGES_REQUESTED** [review]

File: .travis.yml [hunk 2]
```diff
@@ -8,8 +8,7 @@ jdk: openjdk8
 
 env:
   global:
-  - SOURCE_IMAGE_NAME="census-rm-casesvc"
-  - DESTINATION_IMAGE_NAME="$DOCKER_GCP_LOCATION/census-rm-casesvc"
+  - DESTINATION_IMAGE_NAME="eu.gcr.io/census-rm-ci/rm//census-rm-casesvc"
```
Comment (NickGrantX) [inline .travis.yml]: Did you mean to have the double forward-slash in this image name?

**Review by NickGrantX — CHANGES_REQUESTED** [review]
See comment

**Review by AdeHarrison — APPROVED** [review]

**Review by AdamHawtin — APPROVED** [review]

File: .travis.yml [hunk 3]
```diff
@@ -8,8 +8,7 @@ jdk: openjdk8
 
 env:
   global:
-  - SOURCE_IMAGE_NAME="census-rm-casesvc"
-  - DESTINATION_IMAGE_NAME="$DOCKER_GCP_LOCATION/census-rm-casesvc"
+    - DESTINATION_IMAGE_NAME="$DOCKER_GCP_LOCATION/census-rm-casesvc"
```
Comment (NickGrantX) [inline .travis.yml]: spacing

**Review by NickGrantX — APPROVED** [review]
would be good if you could tidy the spacing but no reason to block really

**Pull request merged**